jscomp/others/belt_Option.ml

```diff
@@ -0,0 +1,27 @@
+let getExn = function
+  | Some x -> x
+  | None -> assert false
+
+let fold opt default f = match opt with
+  | Some x -> f x
+  | None -> default
+
+let map opt f = match opt with
+  | Some x -> Some (f x)
+  | None -> None
+
+let flatMap opt f = match opt with
+  | Some x -> f x
+  | None -> None
+
+let getOrElse opt default = match opt with
+  | Some x -> x
+  | None -> default
+
+let exists = function
```

Comment:
Belt.List uses `has`. cc @bobzhang another thing we should unify around. `has` vs `contains` vs `exists`. First one is easier for non-english speakers

Reply:
Actually, not sure why `Belt.List.has` exists